examples/kepler-integration/package.json

```diff
@@ -6,11 +6,14 @@
     "bootstrap": "yarn clean && yarn"
   },
   "dependencies": {
+    "@deck.gl/mapbox": "^8.3.8",
```

Comment:
Since the import for @deck.gl/mapbox is in hubble itself, this needs to be cut/paste the package.json on the root of the repo as a devDependency. `@hubble.gl/react` shouldn't be required since the main `hubble.gl` package already includes it.

`@deck.gl/mapbox` should also be added as a peerDependency to `@hubble.gl/react`'s package.json